pom.xml

```diff
@@ -216,6 +216,7 @@
                 </plugins>
             </build>
         </profile>
+        
```

Comment:
```suggestion
```

If you delete this line, you'll still get a unique git commit hash, and the file won't have actually changed 😈
All you need is a unique commit hash to fool our distribution pipeline, it's fine if every file is identical

but next time: `git commit --allow-empty`

Reply:
Nice, now I don't need to make blank line changes to trigger CIs in the future. I have cut a release by creating a new tag for the same commit, which went through.